pkg/sns/topic.go

```diff
@@ -26,14 +29,14 @@ type Settings struct {
 	Backoff exec.BackoffSettings
 }
 
-type snsTopic struct {
+type SnsTopic struct {
```

Comment:
We could leave this private if we add another interface which also includes `SubscribeSqs` (would be enough to have this in the test only)

Reply:
okay